src/main/resources/category/java/common.xml

```diff
@@ -5,35 +5,26 @@
     <rule name="AvoidCDIReferenceLeak"
           language="java"
           message="Explicit CDI references need to be destroyed otherwise they leak."
-          class="net.sourceforge.pmd.lang.rule.XPathRule"
+          class="net.sourceforge.pmd.lang.rule.xpath.XPathRule"
           typeResolution="true"
           externalInfoUrl="${doc_root}/JavaCodePerformance.md#pml05">
         <description>Problem: A proxy object is created by Contexts and Dependency Injection (CDI) for explicit references, they are not de-referenced implicitly and become a memory leak. &#13;
             Solution: Destroy the reference explicitly.
         </description>
         <priority>1</priority>
         <properties>
-            <property name="tag" value="jpinpoint-rule" type="String" description="for-sonar"/>
-            <property name="tag" value="performance" type="String" description="for-sonar"/>
-            <!--property name="version" value="2.0"/-->
+            <property name="pmd-version" value="7" type="String" description="for-compatibility"/>
```

Comment:
is this property just informational or is used elsewhere?

Reply:
just informational, at least for now